src/modules/todoList.js

```diff
@@ -138,13 +138,14 @@ export default class TodoList {
       dragIcon,
       todo,
       completionIcon,
-      description,
+      description
     );
   }
 
   newMethod5(li, actionIcon, dragIcon, todo, completionIcon, description) {
     const deleteIcon = document.createElement('button');
     deleteIcon.setAttribute('type', 'button');
+    deleteIcon.classList.add('remove');
```

Comment:
- [ ] (OPTIONAL) Kindly also fix the delete tasks function as sometimes I have to refresh the page before a tasks can be deleted successfully 👍